test/integration/sagemaker/test_mnist.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2017-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+from __future__ import absolute_import
+
+import os
+
+import pytest
+from sagemaker.tensorflow import TensorFlow
+
+
+@pytest.mark.skip_gpu
+def test_mnist_cpu(sagemaker_session, ecr_image, instance_type):
+    _run_minist_training(sagemaker_session, ecr_image, instance_type)
+
+
+@pytest.mark.skip_cpu
+def test_mnist_gpu(sagemaker_session, ecr_image, instance_type):
+    _run_minist_training(sagemaker_session, ecr_image, instance_type)
```

Comment:
is there still a need to have these separate? won't a test run just use the appropriate instance type?

Reply:
yeah, you are right. I will merge it.

Reply:
are you planning more tests that will use this `_run_mnist_training` function?